theme.json

```diff
@@ -200,9 +170,9 @@
 			},
 			"core/post-title": {
 				"typography": {
-					"fontFamily": "var(--wp--preset--font-family--system-font)",
+					"fontFamily": "var(--wp--preset--font-family--headings)",
```

Comment:
You have the slug set to `heading` up above, I think we should use that nomenclature, instead of headings

Reply:
Great catch! I fixed that up.